Center the selected item when a select-shaped popup opens (#7067)
ariakit/ariakit: .changeset/200-dialog-initial-focus-scroll.md

"@ariakit/react": patch
---

> Changed [`Dialog`](https://ariakit.com/reference/dialog) to scroll its initially focused element just far enough to become visible, so selected values in [`SelectItem`](https://ariakit.com/reference/select-item) now rest against the edge of long lists instead of being centered.

Author: No need to mention `Select` here.

Author: <!-- ariakit-pr-review review_id=pr-7067-4431961df860fd52c18f77817776077bbd72276e-20260805T095731Z head_sha=4431961df860fd52c18f77817776077bbd72276e -->
Confirmed and still unaddressed at `4431961d`. The line currently reads:

> Changed [`Dialog`](https://ariakit.com/reference/dialog) to scroll its initially focused element just far enough to become visible, so selected values in [`SelectItem`](https://ariakit.com/reference/select-item) now rest against the edge of long lists instead of being centered.

The `SelectItem` clause is also the weakest part of the entry to keep: `Select` does not receive the centering added by this PR, so naming it here advertises only the half of the release that takes something away.

```suggestion
Changed [`Dialog`](https://ariakit.com/reference/dialog) to scroll its initially focused element just far enough to become visible, rather than centering it.
```

**Assessment**

- Severity: Low (verified, evidence: changelog copy only, no runtime effect).
- Priority: P1 (estimate, 90% confidence), because it is a one-line copy change already requested on this thread.
- Breakage likelihood: Likely (verified, evidence: the text ships to the changelog as written).
- Fix effort: Easy (verified, evidence: single line, no code or tests involved).

Author: Addressed in [`af641d55c`](https://github.com/ariakit/ariakit/commit/af641d55c). The `Dialog` changeset is now one user-facing sentence and no longer mentions `SelectItem`.
